src/routes/employeeDb.json

```diff
@@ -0,0 +1,32 @@
+[
```

Comment:
In this JSON, we should still use the same variable format (i.e. camalCase)

Reply:
Update Employee parameters to reflect what we have as data from postGres